Userland/Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -1365,13 +1383,26 @@ ErrorOr<void> StyleComputer::compute_cascaded_values(StyleProperties& style, DOM
                     if (auto iteration_count_value = style.maybe_null_property(PropertyID::AnimationIterationCount); iteration_count_value && iteration_count_value->is_numeric())
                         iteration_count = static_cast<size_t>(iteration_count_value->as_numeric().number());
 
+                    Animation::FillMode fill_mode { Animation::FillMode::None };
+                    if (auto fill_mode_value = style.maybe_null_property(PropertyID::AnimationFillMode); fill_mode_value && fill_mode_value->is_identifier()) {
+                        auto fill_mode_name = fill_mode_value->to_identifier();
+                        if (fill_mode_name == ValueID::Forwards)
+                            fill_mode = Animation::FillMode::Forward;
+                        if (fill_mode_name == ValueID::Backwards)
+                            fill_mode = Animation::FillMode::Backward;
+                        if (fill_mode_name == ValueID::None)
+                            fill_mode = Animation::FillMode::None;
+                        if (fill_mode_name == ValueID::Both)
+                            fill_mode = Animation::FillMode::Both;
+                    }
```

Comment:
Adding the `animation-fill-mode` values to `Enums.json` would get you the to/from-identifier code for free.